.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+/test.md
```

Comment:
Good job adding the `node_modules` folder and `test.md` file here, and using GitHub flow correctly👏